### DISCUSSION TITLE
Frequenza aggiornamento dati sensori temp. aqara

### DISCUSSION THREAD

**Discussion by Matty1184** [indomus/forum#2225] [Hardware & Firmware]
Ciao a tutti e Buon anno!
ho installato dei sensori temperatura Aqara, collegati direttamente tramite in mqtt Zigbee, rilevati tutto ok ma volevo un chiarimento sulla frequenza di aggiornamento. Essendo dispositivi a batteria ovviamente la comunicazione non è continua ma tra i tre dispositivi installati ho discrepanze notevoli.
uno per esempio l'ultimo aggiornamento è 8 minuti fa, l'altro 1 ora fa, l'altro ancora 11 ore fa.
come potete capire si va da una estremità all'altra, c'è un modo per settare questo range di aggiornamento? se si cosa consigliate?

ciao.

**Comment by snakuzzo** [comment]
I sensori si attivano e mandano un aggiornamento della temperatura se c'è una differenza di temperatura (di non ricordo quanti decimi di grado) rispetto all'ultima lettura. Ne consegue che il tempo è variabile...possono passare secondi, così come ore.
Non puoi impostare nulla per forzare questo specifico comportamento...e mi pare pure che sia giusto così, altrimenti le batterie ti durerebbero al massimo due giorni.
Quello che puoi impostare puoi trovarlo [QUI](https://www.zigbee2mqtt.io/devices/WSDCGQ11LM.html)

**Comment by Matty1184** [comment]
Come pensavo..  mi viene però allora questo dubbio.
se uno di questi dispositivi è fermo da giorni sugli sessi valori... e sono sicurissimo che la variazione di temperatura o umidità c'è stata (è un bagno con doccia aperta) come mai non legge? 
nella funzione mappa di Zigbeeg2Mqtt lo vedo collegato (se si fosse scollegato non vedrei l'associazione giusto?).

**Reply by snakuzzo** [reply]
>  come mai non legge?

e io che ne so ? Mica ho la palla di vetro?

> nella funzione mappa di Zigbeeg2Mqtt lo vedo collegato (se si fosse scollegato non vedrei l'associazione giusto?).

no...non lo vedresti "scollegato". Lo continui a vedere lì

**Reply by snakuzzo** [reply]
P.S. Non creare nuovi thread (come hai appena fatto)...rispondi sempre in risposta all'ultimo messaggio

**Reply by Matty1184** [reply]
> P.S. Non creare nuovi thread (come hai appena fatto)...rispondi sempre in risposta all'ultimo messaggio

OPS. scusami!

> no...non lo vedresti "scollegato". Lo continui a vedere lì

ok lo vedrei però solo come dispositivo senza la linea di collegamento con il coordinator  oppure con la linea?
io ora lo vedo con la linea verde di connessione. la linea verde di connessione io l'ho interpretata come "connesso"
altra particolarità, al riavvio di HA il dato di lettura risale a 1 minuto fa, ma è sempre agli stessi valori bloccati da giorni.

**Reply by snakuzzo** [reply]
non c'entra niente quello che vedi lì...quello risale all'ultimo collegamento che c'è stato.
Per vedere la disponibilità di un device potresti provare a vedere [QUI](https://www.zigbee2mqtt.io/guide/configuration/device-availability.html)

**Reply by Matty1184** [reply]
Grazie per la funzione, non ne ero a conoscenza e molto utile.
ho verificato comunque i log, e onestamente ogni tanto (dico ogni tanto) il dato arriva ma errato.
il mio ogni tanto è sicuramente dato dal fatto che se come mi hai spiegato il dispositivo manda i dati ogni variazione di x,x gradi automaticamente avendo in pancia il dato errato e stabile, per lui non è necessario inviarlo presumo.
provo a cambiare la pila, non vorrei fosse quelle che mi falsa tutto.

**Reply by Matty1184** [reply]
aggiormento. ho verificato che il punto in cui ho collocato il dispositivo non è coperto perfettamente dal segnale zigbee.
